app/src/test/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModelTest.kt

```diff
@@ -0,0 +1,72 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import app.cash.turbine.test
+import co.nimblehq.compose.crypto.domain.usecase.GetMyCoinsUseCase
+import co.nimblehq.compose.crypto.test.MockUtil
+import co.nimblehq.compose.crypto.ui.screens.BaseViewModelTest
+import co.nimblehq.compose.crypto.ui.uimodel.toUiModel
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.*
+
+@ExperimentalCoroutinesApi
+class HomeViewModelTest : BaseViewModelTest() {
```

Comment:
I appreciate you for adding the `Unit Test.` 👏  but I think It would be better if we add it later after creating the setup or updating the base class (`BaseViewModelTest`). So we can get rid of adding dependencies of `Unit Test` in `build.gradle`. What do you think? 🙇‍♀️

Reply:
OK, Let's see on next integration  `https://github.com/nimblehq/jetpack-compose-crypto/pull/27#discussion_r963652707`

Reply:
@Wadeewee I don't like having business logic without tests. The current VM and networking layers are good enough for this compose project. Hence we should have unit tests.

Reply:
I totally agree with you, @luongvo, but It seems like the base test case needs to be updated. That's why I think we can skip the `Unit Test` for this PR because It has a lot of files that need to focus on e.g. `CoroutineTestRule` 🙏